src/Symfony/Component/Routing/Tests/RouteCollectionTest.php

```diff
@@ -362,5 +362,19 @@ public function testAddWithPriority()
         ];
 
         $this->assertSame($expected, $collection2->all());
+
+        $collection3 = new RouteCollection();
```

Comment:
please create a separate test function. this test does not need to be in the same test case